lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -25,6 +26,9 @@ abstract class DoctrineCommand extends Command
     /** @var DependencyFactory|null */
     private $dependencyFactory;
 
+    /** @var StyleInterface */
+    protected $io;
```

Comment:
should be private

Reply:
Can't be private. It is used by other commands which extend from `DoctrineCommand`.